pkgs/top-level/perl-packages.nix

```diff
@@ -804,6 +804,21 @@ let
     };
   };
 
+  AstroFITSHeader = buildPerlModule rec {
+    pname = "Astro-FITS-Header";
+    version = "3.07";
+    src = fetchurl {
+      url = "mirror://cpan/authors/id/T/TJ/TJENNESS/${pname}-${version}.tar.gz";
+      sha256 = "530d59ef0c0935f9862d187187a2d7583b12c639bb67db14f983322b161892d9";
+    };
+    buildInputs = [ ModuleBuild ];
```

Comment:
`buildPerlModule` already has `ModuleBuild`

```suggestion
```

Need to wait to merge util https://github.com/NixOS/nixpkgs/pull/97098/commits/340f23f171b13875263881b637d6924dcc5045d7 hits `master`